server/routes/service.js

```diff
@@ -1,3 +1,131 @@
-const transport = require('../routes/transport');
-const db = require('../db/db'); 
+require('dotenv').config();
+const bcrypt = require('bcrypt');
 
+const saltRounds = 10;
+const jwt = require('jsonwebtoken');
+const db = require('../db/db');
+
+// Check if the user is already authenticated
+function checkAuthStatus(authCookie) {
+  jwt.verify(authCookie, process.env.ACCESS_TOKEN_SECRET, (err) => {
```

Comment:
I wonder if this function has an async/await method.  Currently, I think you're returning `403` within the callback function and but not within the outerscope, so checkAuthStatus might always return `undefined`